libkbfs/backpressure_disk_limiter.go

```diff
@@ -812,20 +854,31 @@ func (bdl *backpressureDiskLimiter) onJournalDisable(
 	defer bdl.lock.Unlock()
 	bdl.journalTracker.onDisable(
 		journalStoredBytes, journalUnflushedBytes, journalFiles, chargedTo)
+	bdl.overallByteTracker.onDisable(journalStoredBytes)
 }
 
-func (bdl *backpressureDiskLimiter) onDiskBlockCacheEnable(ctx context.Context,
-	diskCacheBytes int64) {
+func (bdl *backpressureDiskLimiter) onByteTrackerEnable(ctx context.Context,
```

Comment:
Again, the naming here and below seems to imply that the journal would a valid type.